src/gctools/cscript.lisp

```diff
@@ -10,7 +10,7 @@
            #~"exposeClasses3.cc"
            #~"boehmGarbageCollection.cc"
            #~"mmtkGarbageCollection.cc"
-           #~"mpsGarbageCollection.cc"
+           ;#~"mpsGarbageCollection.cc"
```

Comment:
If we don't ever need it we should just delete this file. MMTk works differently from MPS so I don't think it will be much help as a reference, and even if we do need it, the file will always be there in the git history.